### PR TITLE
Created issue and pull request template

### PR DESCRIPTION
**Pull request by itz-deepak**
Please Review my PR

- Issue #2

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Issue Number:
+- Here goes the issue number with a #
+-----------------------------------------------------------
+## Issue Description:
+- Describe the issue fixed here
+-----------------------------------------------------------
+## Checklist:
```
Comment (nikhil25803) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:7]: In the `checklist` section, 
+ Tested the endpoint on sample data.
+ No conflicts
+ Have mentioned the endpoint in the `ENDPOINTS.md` file.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+-----------------------------------------------------------
+## Issue Description:
+- Describe the issue fixed here
+-----------------------------------------------------------
+## Checklist:
+- [ ] Testing
+- [ ] No conflicts with the main branch
+-----------------------------------------------------------
+## Screenshot of the changes:
+Earlier | Current
```
Comment (nikhil25803) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:12]: Rename the table as a python script snapshot and sample output screenshot.

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Prerequisites
```
Comment (nikhil25803) [inline .github/ISSUE_TEMPLATE/issue_template.md:1]: Modify the issue templates as
+ Which endpoint you are willing to add?
+ Reference formula for the calculation.
+ Name of the endpoint you want to suggest.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Requested so changes to make the issue and the PR template more confined and contextual. Please go through it and make changes accordingly.

**Comment by itz-deepak** [comment]
I have applied the changes suggested please review

**Review by nikhil25803 — APPROVED** [review]
Changes approved

**Comment by nikhil25803** [comment]
Congrats 🎉, your PR got merged. To be eligible for the swags from ClueLess. Follow these steps.

1. Star this repository. ⭐
2. SignIn / SignUp in [[ClueLess Website](https://www.clueless.tech/)](https://www.clueless.tech/)
3. Go to the profile section from the top right-hand corner.



Comment your profile URL under this comment tagging the person who merged your PR.

That’s it. Follow the leaderboard!

And don’t forget to share your achievement with your friends.

**Pull request merged**